lab_1/main.py

```diff
@@ -1,94 +1,121 @@
-"""
-Lab 1
-A concordance extraction
```

Comment:
Please, leave lab description and descriptions of the functions as they are.

Reply:
Changed. Thanks.